client-rpc/src/main.rs

```diff
@@ -37,6 +37,15 @@ pub(crate) struct Options {
     )]
     network_id: String,
 
+    #[structopt(
+        name = "network-type",
+        short = "i",
+        long,
+        default_value = "test",
```

Comment:
I think the default value should be `dev`

Reply:
ok